app/components/RegionDropdown/RegionDropdown.module.scss

```diff
@@ -1,75 +1,59 @@
 @import "../../styles/colors";
+@import "../../styles/vars";
+@import "../../styles/typography";
+
+// Focus-ring (move somewhere more global later)
+@mixin focus-ring {
+  box-shadow: 0 0 0 2px var(--control-focus-ring);
+}
 
 .dropdown-container {
   position: relative;
   cursor: pointer;
-  width: 16.25rem;
+  width: calc($space-3xl * 7);
```

Comment:
I don't particularly like how this is a fixed width, but I couldn't for the life of me figure out how to make the width dynamic based on the trigger text width.

Reply:
AFAIK its not possible with native select, will need to move to a custom dropdown for that behaviour

Reply:
Why is this a calc tho? Should it be part of the spacing scale?

Reply:
It was my attempt at adding _some_ meaning to the value... 😅 

I don't think it should be part of the spacing scale because it's pretty bespoke and not really used in the same context as the other spacing values. We could add like "standard control size" variables or something?

Reply:
Na I reckon just hard code it here - if we come to creating a standard set we can loop back round?

Could always make a local variable so it's actually named which gives it some meaning 🤔

Reply:
donezo